### PR TITLE
Update Google Analytics

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description

Update Google Analytics.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by thepsalmist — COMMENTED** [review]
Can we set this `GA_TRACKING_ID` as an env variable @kelvinkipruto

**Comment by kelvinkipruto** [comment]
> Can we set this `GA_TRACKING_ID` as an env variable @kelvinkipruto

@thepsalmist It will be public anyway, and there is only one place it is used.

**Comment by kilemensi** [comment]
> > Can we set this `GA_TRACKING_ID` as an env variable @kelvinkipruto
> 
> @thepsalmist It will be public anyway, and there is only one place it is used.

What he means @kelvinkipruto is changing the ID value shouldn't require code change; just updating the environment variable on Docker/Dokku is all it should take.

File: src/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_APP_URL = 'https://sensors.africa/'
+NEXT_PUBLIC_GA_ID =  'G-8M50XTMKVG'
```
Comment (kilemensi) [inline src/.env:2]:
```suggestion
NEXT_PUBLIC_APP_URL = 'http://localhost:3000'
```
[`.env`](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#environment-variable-load-order) is for default values only i.e. when it doesn't matter if it's PROD or DEV or local machine.

- PROD domain should only be set in PROD (via dokku config:set)
- If we set GA_ID here, it means even in DEV (think Vercel) this ID will be used & hence will pollute the actual data. Therefore, it's best to also set it in PROD.

PS: Lets standardise how we name this variable `GA_ID` here, `GOOGLE_ANALYTICS_ID` in ui repo, etc., makes it harder to remember.

**Review by thepsalmist — APPROVED** [review]
LGTM @kelvinkipruto

**Review by kilemensi — COMMENTED** [review]
👍🏽

**Comment by kelvinkipruto** [comment]
@thepsalmist When deploying to dokku, add the `GOOGLE_ANALYTICS_ID` as an environment variable with the correct value

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_APP_URL = 'http://localhost:3000';
```
Comment (kilemensi) [inline .env:1]:
```suggestion
NEXT_PUBLIC_APP_URL = 'http://localhost:3000';

```
New line

File: .env.sample [hunk 3]
```diff
@@ -1 +1,2 @@
 NEXT_PUBLIC_APP_URL = 'http://localhost:3000'
+GOOGLE_ANALYTICS_ID = 'UA-XXXXXXXXX-X'
```
Comment (kilemensi) [inline .env.sample:2]:
```suggestion
GOOGLE_ANALYTICS_ID = 'UA-XXXXXXXXX-X'

```
New line

**Review by kilemensi — APPROVED** [review]
🚀

**Pull request merged**